meter/tasmota/connection.go

```diff
@@ -200,21 +201,31 @@ func (c *Connection) CurrentPower() (float64, error) {
 	if err != nil {
 		return 0, err
 	}
-	var res float64
+	var combinedPower float64
 	for _, channel := range c.channels {
 		power, err := s.StatusSNS.Energy.Power.Channel(channel)
 		if err != nil {
 			return 0, err
 		}
-		res += power
+		combinedPower += power
 	}
-	return res, nil
+	// add SML power
```

Comment:
Why is this an addition and not either or?

Reply:
oh that's right